conf/cms/secrets.sample.py

```diff
@@ -1,6 +1,4 @@
-# After https://jira.tacc.utexas.edu/browse/COOKS-21, then should
-#be route to frontend: "/protx/demographics"
-LOGIN_REDIRECT_URL = "/protx/api/maltreatment"
+LOGIN_REDIRECT_URL = "/demographics"
```

Comment:
Shouldn't this also be `/analytics` so we are redirecting users to the same page after login as after onboarding complete?

Reply:
👍 It should be to the same spot like you said.  but now I'm uncertain what is the correct spot.  It might be /demographics or /analytics 🤷‍♂️   RIght now in pprd, we generic login going to "demographics" while the onboarding button goes to "protx" -> which goes to analytics 🤷‍♂️ .  

Pick one now and ask Tracy at next meeting?

Reply:
Go with analytics. I'll take the blame me if she doesn't like it. =)